compiler/damlc/daml-ide-core/dlint.yaml

```diff
@@ -276,13 +246,13 @@
     - warn: {lhs: f (fst p) (snd p), rhs: uncurry f p}
     - warn: {lhs: ($) . f, rhs: f, name: Redundant $}
     - warn: {lhs: (f $), rhs: f, name: Redundant $}
-    - warn: {lhs: (Data.Function.& f), rhs: f, name: Redundant Data.Function.&}
```

Comment:
Afaik we don’t have `&` in DAML. I guess this rule doesn’t do any harm in that case but it’s a bit confusing.

Reply:
Ok